File: service/api/src/main/java/org/delivery/api/common/annotation/Business.java
package org.delivery.api.common.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

import org.springframework.core.annotation.AliasFor;
import org.springframework.stereotype.Service;

@Service
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.TYPE)
public @interface Business {

	@AliasFor(annotation = Service.class)
	String value() default "";

}

File: delivery/common/src/main/kotlin/org/delivery/common/annotation/Business.kt
package org.delivery.common.annotation

import org.springframework.core.annotation.AliasFor
import org.springframework.stereotype.Service

@Service
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.CLASS)
annotation class Business(
    @get:AliasFor(annotation = Service::class)
    val value: String = ""
)
